std_msgs/Header headder
sensor_suite/LabeledBoundingBox2D[] boxes
